src/Temporalio/Worker/Tuning/SlotPermit.cs

```diff
@@ -0,0 +1,33 @@
+namespace Temporalio.Worker.Tuning
+{
+    /// <summary>
+    /// A permit to use a slot for a workflow/activity/local activity task.
```

Comment:
Can tell users they can extend this class if they want (unlike what we allowed in Java)